app/services/market_service.rb

```diff
@@ -1,14 +0,0 @@
-class MarketService
```

Comment:
I think we'll want to refactor and bring this back for maintainability's sake in the future.